src/Uno.UI.RemoteControl.DevServer.Tests/Helpers/SolutionHelper.cs

```diff
@@ -4,18 +4,22 @@
 
 namespace Uno.UI.RemoteControl.DevServer.Tests.Helpers;
 
+#pragma warning disable VSTHRD002 // Async methods must run in sync context here
+
 public class SolutionHelper : IDisposable
 {
+	private readonly TestContext _testContext;
 	private readonly string _solutionFileName;
 	private readonly string _tempFolder;
 
 	public string TempFolder => _tempFolder;
 	public string SolutionFile => Path.Combine(_tempFolder, _solutionFileName + ".sln");
 
-	private bool isDisposed;
+	private bool _isDisposed;
 
-	public SolutionHelper(string solutionFileName = "MyApp")
+	public SolutionHelper(TestContext testContext, string solutionFileName = "MyApp")
 	{
+		_testContext = testContext;
```

Comment:
Consider:

```suggestion
		_testContext = testContext;
		Assert.IsFalse(string.IsNullOrEmpty(_testContext.TestName));
```

This would ensure that the TestContext being passed here is from a test, and not from class/assembly init/cleanup. Note, that assert sill always pass in v3, but will start to have the intended meaning in v4, but it's still safe to do it now.